### PR TITLE
Test api endpoints MERGE PROPERLY

### PR DESCRIPTION
**Pull request by jacobrs**
Need to merge this PR correctly

### CODE REVIEW

File: server/controllers/message.controller.js [hunk 1]
```diff
@@ -32,12 +32,12 @@ export function getAllMessages(req, res) {
   });
 }
 
-export function getMessagesByStudyGroup(req, res) {
-  Message.find({ studyGroup: req.params.studyGroup }).exec((err, messages) => {
+export function getMessagesFromStudyGroup(req, res) {
```
Comment (jacobrs) [inline server/controllers/message.controller.js]: This line broke the chat loading history.

**Review by jacobrs — COMMENTED** [review]
Needs fixing

**Review by simon-bourque — APPROVED** [review]
We already approved this before right, its just a matter of merging it correctly

**Comment by ONourry** [comment]
Can't test right now but make sure the messages work fine this time before merging

**Pull request merged**